app/controllers/items_controller.rb

```diff
@@ -45,7 +45,7 @@ def create
   def search
```

Comment:
Should this code be in the ItemsController or the Item model class? I always thought things like search were part of the business logic - usually controllers only have the CRUDI operations. Let me know!